pkg/uploader/provider/provider.go

```diff
@@ -48,6 +48,7 @@ type Provider interface {
 		tags map[string]string,
 		forceFull bool,
 		parentSnapshot string,
+		volMode uploader.PersistentVolumeMode,
 		updater uploader.ProgressUpdater) (string, bool, error)
 	// RunRestore which will do restore for one specific volume with given snapshot id and return error
 	// updater is used for updating backup progress which implement by third-party
```

Comment:
Need to pass `volMode uploader.PersistentVolumeMode` in `RunRestore as well

Reply:
updated